Sources/Components/Button/MapSettings/MapSettingsButton.swift

```diff
@@ -0,0 +1,100 @@
+//
+//  Copyright © 2019 FINN.no. All rights reserved.
+//
+
+import UIKit
+
+public protocol MapSettingsButtonDelegate: AnyObject {
+    func mapSettingsButtonDidSelectChangeMapTypeButton(_ view: MapSettingsButton)
+    func mapSettingsButtonDidSelectCenterMapButton(_ view: MapSettingsButton)
+}
+
+public final class MapSettingsButton: UIView {
```

Comment:
Could you rename this to MapSettingsView? Button confused me a bit..

Reply:
Yes, I can do that. I named it `MapSettingsButton` since I thought it was a convention due to all the other buttons within `Components/Button` being named `<name>Button.swift`

Reply:
I didn't realise you put this in the `Button` folder. Since this view is a bit more complicated than a single button I would rename it and put it under `Components` ;)